google-cloud-spanner/src/test/java/com/google/cloud/spanner/DefaultNumChannelsTest.java

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Not sure about this, but can we check if there are existing tests around default number of channels, and make changes to that file itself instead of creating a new file?

Reply:
We can add it to `SpannerOptionsTest.java` file.